test/gtest/uct/test_amo_add_xor.cc

```diff
@@ -47,6 +49,15 @@ UCS_TEST_SKIP_COND_P(uct_amo_add_xor_test, add32,
     test_op<uint32_t, UCT_ATOMIC_OP_ADD>(add_op<uint32_t>);
 }
 
+// Make sure registration has atomic capability, and non supporting transports
+// perform anyways
+UCS_TEST_SKIP_COND_P(uct_amo_add_xor_test, addr32_symmetric_rkey,
+                     !check_atomics(UCS_BIT(UCT_ATOMIC_OP_ADD), OP32))
```

Comment:
should we also check symmetric key support?

Reply:
I think we want to test that it works both with a symmetric key and without it, when the symmetric flag was passed. (exhaustion or no support).